core/eval.go

```diff
@@ -197,6 +197,29 @@ func evalSET(args []string) []byte {
 	return RESP_OK
 }
 
+// evalMSET puts multiple <key, value> pairs in db as in the args
+// args must contain key and value pairs.
+
+// Returns encoded error response if at least a <key, value> pair is not part of args
+// Returns encoded OK RESP once new entries are added
+// If the key already exists then the value will be overwritten and expiry will be discarded
```

Comment:
nit: since there is no expiry, we can remove `expiry will be discarded`

Reply:
When a key is set using `SET` with expiry then overwritten with `MSET`. Then in this case, the expiry will be discarded right?

Reply:
You are right. I missed the case. Thanks.